From 4f51e1a9b7583cb8e4b1cd85556b841837a2b7eb Mon Sep 17 00:00:00 2001
From: hazveg <133221043+hazveg@users.noreply.github.com>
Date: Tue, 25 Jun 2024 13:54:25 +0200
Subject: [PATCH] This patch applies catppuccin colors to slock

Doesn't follow any specific catppuccin. Mostly follows mocha, but
mocha's red looks too pink for my liking; so i think i used latte's.
---
 config.def.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/config.def.h b/config.def.h
index 9855e21..b262c9f 100644
--- a/config.def.h
+++ b/config.def.h
@@ -3,9 +3,9 @@ static const char *user  = "nobody";
 static const char *group = "nogroup";
 
 static const char *colorname[NUMCOLS] = {
-	[INIT] =   "black",     /* after initialization */
-	[INPUT] =  "#005577",   /* during input */
-	[FAILED] = "#CC3333",   /* wrong password */
+	[INIT] =   "#11111b",     /* after initialization */
+	[INPUT] =  "#a6e3a1",   /* during input */
+	[FAILED] = "#d20f39",   /* wrong password */
 };
 
 /* treat a cleared input like a wrong password (color) */
-- 
2.43.0

